src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+type TodoFormProps = {
+  onSubmit: (title: string) => void;
+  disabled: boolean;
+};
+
+export const TodoForm: React.FC<TodoFormProps> = ({ onSubmit, disabled }) => {
```

Comment:
It's a good practice to use React.memo for the functional components to potentially improve the overall performance. Wrap your component with React.memo like this: `export const TodoForm = React.memo<TodoFormProps>(...)`.